docs/schemas/itemMeta.json

```diff
@@ -0,0 +1,996 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "tas-item",
+  "description": "A question item for TAS",
+  "definitions": {
+
+    "tas-singleLine": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-singleLine",
+      "title": "tas-string",
+      "description": "A text question item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["singleLine"]
+        },
+        "minSize": {
+          "description": "Min number of characters",
+          "type": "number",
+          "default": 1
```

Comment:
So far I haven't been using default at all in the schemas. We don't have any known tooling that could make use of it, so would prefer to not start now (let me know if I've missed something!)